builtin/providers/google/config.go

```diff
@@ -7,9 +7,12 @@ import (
 	"net/http"
 	"os"
 
-	"code.google.com/p/goauth2/oauth"
-	"code.google.com/p/goauth2/oauth/jwt"
 	"code.google.com/p/google-api-go-client/compute/v1"
+	// oauth2 "github.com/rasa/oauth2-fork-b3f9a68"
+	"github.com/rasa/oauth2-fork-b3f9a68"
+
+	// oauth2 "github.com/rasa/oauth2-fork-b3f9a68/google"
+	"github.com/rasa/oauth2-fork-b3f9a68/google"
```

Comment:
I am not sure.  I tried github.com/golang/oauth2/google and couldn't work out how to make the build systems pull in the dependency.  When I saw packer using the other one I just copied that and it worked.  I'd like to know the full story as well.